communications/ff_msgs/msg/PerchState.msg

```diff
@@ -23,33 +23,33 @@ std_msgs/Header header
 # Feedback
 int8 state
 
-int8 RECOVERY_MOVING_TO_RECOVERY_POSE   = 18
-int8 RECOVERY_SWITCHING_TO_ML_LOC       = 17
-int8 RECOVERY_STOWING_ARM               = 16
-int8 RECOVERY_MOVING_TO_APPROACH_POSE   = 15
-int8 RECOVERY_OPENING_GRIPPER           = 14
-int8 INITIALIZING                       = 13
-int8 UNKNOWN                            = 12
-int8 PERCHING_MAX_STATE                 = 11
-int8 PERCHING_SWITCHING_TO_HR_LOC       = 11
-int8 PERCHING_MOVING_TO_APPROACH_POSE   = 10
-int8 PERCHING_ENSURING_APPROACH_POSE    = 9
-int8 PERCHING_DEPLOYING_ARM             = 8
-int8 PERCHING_OPENING_GRIPPER           = 7
-int8 PERCHING_MOVING_TO_COMPLETE_POSE   = 6
-int8 PERCHING_CLOSING_GRIPPER           = 5
-int8 PERCHING_CHECKING_ATTACHED         = 4
-int8 PERCHING_WAITING_FOR_SPIN_DOWN     = 3
-int8 PERCHING_SWITCHING_TO_PL_LOC       = 2
-int8 PERCHING_STOPPING                  = 1
-int8 PERCHED                            = 0
+int8 RECOVERY_MOVING_TO_RECOVERY_POSE   = 20
+int8 RECOVERY_SWITCHING_TO_ML_LOC       = 19
+int8 RECOVERY_STOWING_ARM               = 18
+int8 RECOVERY_MOVING_TO_APPROACH_POSE   = 17
+int8 RECOVERY_OPENING_GRIPPER           = 16
+int8 INITIALIZING                       = 15
+int8 UNKNOWN                            = 14
+int8 PERCHING_MAX_STATE                 = 13
+int8 PERCHING_SWITCHING_TO_HR_LOC       = 12
+int8 PERCHING_MOVING_TO_APPROACH_POSE   = 11
+int8 PERCHING_ENSURING_APPROACH_POSE    = 10
+int8 PERCHING_DEPLOYING_ARM             = 9
+int8 PERCHING_OPENING_GRIPPER           = 8
+int8 PERCHING_MOVING_TO_COMPLETE_POSE   = 7
+int8 PERCHING_CLOSING_GRIPPER           = 6
+int8 PERCHING_CHECKING_ATTACHED         = 5
+int8 PERCHING_WAITING_FOR_SPIN_DOWN     = 4
+int8 PERCHING_SWITCHING_TO_PL_LOC       = 3
+int8 PERCHING_STOPPING                  = 2
+int8 PERCHED                            = 1
```

Comment:
These values are based on the mobility state that Lorenzo defined a long time ago and docking matches. Basically, we have N to 1 as perching, 0 as perched, and -1 to -N as unperching. If you change unperch to 0, the executive/GDS will display that Astrobee is perched when Astrobee is unperched. If there is a reason unperched needs to be 0 then I will have to change the executive to map the values correctly. If there isn't a reason, please change them back to what they were.

Reply:
Thank you! I didn't know about that convention, then unperch can be -1, and still something easy to remember